qa-tests-backend/src/test/groovy/ImageScanningTest.groovy

```diff
@@ -492,6 +492,7 @@ class ImageScanningTest extends BaseSpecification {
     @Category(Integration)
     def "Verify image scan exceptions - #scanner.name() - #testAspect"() {
         Assume.assumeTrue(scanner.isTestable())
+        Assume.assumeFalse(testAspect == "missing required registry" && Env.mustGetOrchestratorType() == OrchestratorTypes.OPENSHIFT)
```

Comment:
How about using `@IgnoreIf` from spock. `Assume` comes from junit and was changed in latest version making it harder to upgrade for us.